libs/components/ag-grid/src/lib/modules/ag-grid/types/autocomplete-properties.ts

```diff
@@ -2,20 +2,62 @@ import { TemplateRef } from '@angular/core';
 import { SkyAutocompleteSelectionChange } from '@skyux/lookup';
 
 export interface SkyAgGridAutocompleteProperties {
+  /**
+   * Allows the user to specify arbitrary values not in the search results.
+   * @default false
+   */
   allowAnyValue?: boolean;
+  /**
+   * The static data source for the autocomplete cell to search when users enter text. For a dynamic data source such as an array that changes due to server calls, use search instead.
+   */
   data?: unknown[];
+  /**
+   * How many milliseconds to wait before searching while users enter text in the autocomplete field.
+   * @default 0
+   */
   debounceTime?: number;
+  /**
+   * The object property to display in the text input after users select an item in the dropdown list.
+   * @default "name"
+   */
   descriptorProperty?: string;
+  /**
+   * Highlights the search text in each search result. Set this to false when your search finds results that are not exact text matches, e.g. returning "Bob" for the term "Robert."
+   * @default true
+   */
   highlightSearchText?: boolean;
+  /**
+   * The object properties to search.
+   * @default ["name"]
+   */
   propertiesToSearch?: string[];
+  /**
+   * The function that dynamically manages the data source when users change the text in the autocomplete cell. The search function must return an array or a promise of an array.
```

Comment:
Is this correct? I might just not be connecting the dots correctly, but "dynamically manages the data source when users change the text in the autocomplete cell" doesn't sound like what I'd expect a "search" function to do.

Reply:
this is pulled from the autocomplete/lookup docs and the phrasing is a little odd, but from what I can tell it just means that it provides the search results when the user changes the text, and that those results are then displayed as the "data" in the component

Reply:
```suggestion
   * The function that dynamically manages the data to display in search results when users change the text in the autocomplete cell. The search function must return an array or a promise of an array.
```
Do you think this rewrite is any clearer? I think it's "data source" more than anything else that was throwing me off, and I _think_ this rewrite makes things a little clearer. ... Or maybe I've just been looking at this too long. I'm going to hold off on making this suggestion in the corresponding property for lookup until you have a chance to respond here.